front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -56,21 +70,21 @@ export const formatSuggestedViasToRowVias = (
     const { arrival, onStopSignal, stopFor, theoreticalMargin } = objectToUse || {};
 
     const isMarginValid = theoreticalMargin ? marginRegExValidation.test(theoreticalMargin) : true;
-    let departure: string | undefined;
-    if (stopFor) {
-      if (i === 0) {
-        departure = startTime
-          ? secToHoursString(datetime2sec(new Date(startTime)) + Number(stopFor), true)
-          : undefined;
-      } else if (arrival) {
-        departure = secToHoursString(time2sec(arrival) + Number(stopFor), true);
-      }
-    }
+    const roundedArrivalTime = i === 0 ? 'PT0S' : arrival;
```

Comment:
Maybe it's clearer like that

```suggestion
    const durationArrivalTime = i === 0 ? 'PT0S' : arrival;
```

Reply:
done